crates/biome_formatter/src/builders.rs

```diff
@@ -2642,9 +2642,15 @@ impl<'a, Context> BestFitting<'a, Context> {
     ///
     /// You're looking for a way to create a `BestFitting` object, use the `best_fitting![least_expanded, most_expanded]` macro.
     ///
+    /// This is intended to be only used in the `best_fitting!` macro. As we can't place tail
+    /// expressions in a block for temporary lifetime extension since Rust 2024, we can't use an
+    /// `unsafe` block in the macro. Thus, this function can't be marked as unsafe, but it shouldn't
+    /// be used from outside.
+    ///
     /// ## Safety
     /// The slice must contain at least two variants.
```

Comment:
Is the safety part still valid? Asking because we don't have an unsafe block anymore

Reply:
> We can remove it because it no longer appear in the docs though

Please keep it. Safety notes are important for internal understanding too, not just the docs.